webextensions/browser-compat-data.json

```diff
@@ -1212,19 +1215,24 @@
       }, 
       "Firefox": {
         "notes": [
-          "Only the 'basic' type is supported."
+          "Only the 'basic' type is supported.",
+          "For the 'basic' type, 'iconUrl' is optional."
         ], 
         "support": "45.0"
       }, 
       "Firefox for Android": {
         "notes": [
-          "Only the 'basic' type is supported."
+          "Only the 'basic' type is supported.",
+          "For the 'basic' type, 'iconUrl' is optional."
         ], 
         "support": "48.0"
       }
     }, 
     "create": {
       "Chrome": {
+        "notes": [
+          "For the 'basic' type, if 'iconUrl' is present, but the URL can not be loaded, the notification will not be displayed, an error will be shown in the extension's background page console and 'runtime.lastError' will be set in the callback function."
+        ],
```

Comment:
I think this is just saying the same thing as "iconUrl is mandatory", which is already stated in the main body of the page.